android/brave_java_sources.gni

```diff
@@ -295,7 +352,6 @@ brave_java_sources = [
   "../../brave/android/java/org/chromium/chrome/browser/partnercustomizations/CloseBraveManager.java",
   "../../brave/android/java/org/chromium/chrome/browser/password_manager/settings/BraveExportFlow.java",
   "../../brave/android/java/org/chromium/chrome/browser/password_manager/settings/BravePasswordSettingsBase.java",
-  "../../brave/android/java/org/chromium/chrome/browser/playlist/PlaylistHostActivity.java",
```

Comment:
why does this go away? There should not be any code changes in this PR except for namespace changes for the kt code